settei/utils.py

```diff
@@ -0,0 +1,14 @@
+import importlib
+
+
+def import_hook(module_path: str):
```

Comment:
Can be replaced with `pydoc.locate()`

Reply:
`pydoc.locate()` only accepts the form of `module.submodule.attr`, but I think the form of `module.submodule:attr` is way better. And `pydoc.locate` has no available official documentation neither example.